src/backend/src/controllers/teams.controllers.ts

```diff
@@ -50,4 +50,15 @@ export default class TeamsController {
       next(error);
     }
   }
+
+  static async setTeamHead(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { userId } = req.body;
+      const user = await getCurrentUser(res);
+      const team = await TeamsService.setTeamHead(user, req.params.teamId, userId);
```

Comment:
can you parse out the teamId so like {teamId} = req.params, just for the sake of consistency with other endpoints